gitops/generate.go

```diff
@@ -85,6 +87,31 @@ func Generate(fs afero.Afero, gitOpsFolder string, outputFolder string, componen
 	return GenerateParentKustomize(fs, gitOpsFolder, commonStorage)
 }
 
+// GenerateOverlays generates the overlays dir during the binding reconcile
```

Comment:
Nit: Don't refer to the binding reconcile here, maybe just something like "GenerateOverlays generates the overlays dir from a given BindingComponent"?